jackson-modules/jackson-conversions-3/src/main/java/com/baeldung/jackson/absentvsnull/SampleUtils.java

```diff
@@ -0,0 +1,100 @@
+package com.baeldung.jackson.absentvsnull;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import com.baeldung.jackson.absentvsnull.model.Sample;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class SampleUtils {
+
+    public static final TypeReference<HashMap<String, Serializable>> MAP_TYPE = new TypeReference<HashMap<String, Serializable>>() {
+    };
+    public static final ObjectMapper MAPPER = new ObjectMapper();
+
+    static {
+        MAPPER.enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES);
+
+        MAPPER.setSerializationInclusion(Include.NON_NULL);
+        MAPPER.setSerializationInclusion(Include.NON_ABSENT);
+        MAPPER.setSerializationInclusion(Include.NON_DEFAULT);
+    }
+
+    private SampleUtils() {
+    }
+
+    public static Sample deserialize(String json) throws JsonProcessingException {
+        return MAPPER.readValue(json, Sample.class);
+    }
+
+    public static Map<String, Serializable> deserializeAsMap(String json) throws JsonProcessingException {
```

Comment:
I think this method could be removed. There are many places in the test that simply deserialize directly. Otherwise, this is only used in `updateNonAbsent`.